src/utils/plural.js

```diff
@@ -3,10 +3,51 @@ import i18n from '../i18n'
 // Pluralization l10n http://docs.translatehouse.org/projects/localization-guide/en/latest/l10n/pluralforms.html?id=l10n/pluralforms
 export function declOfNum (n, titles) {
   switch (i18n.name) {
```

Comment:
Where is i18n variable coming from?

Reply:
import i18n from '../i18n'
in first line